README.md

```diff
@@ -1,8 +1,8 @@
-## loopback-connector-postgresql
+# loopback-connector-postgresql
 
-The PostgreSQL Connector module for for [loopback-datasource-juggler](http://docs.strongloop.com/loopback-datasource-juggler/).
+The PostgreSQL Connector module for [loopback-datasource-juggler](http://docs.strongloop.com/loopback-datasource-juggler/).
```

Comment:
**Please Note:**

I could not find the equivalent of http://docs.strongloop.com/loopback-datasource-juggler/ in http://loopback.io/doc/en/lb2/

Could you help?